packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,2982 @@
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+      "type": "CUSTOM",
+      "role": "cr0Yq6IJxGIr0ouum0g3",
+      "resource-set": "iamoJDFKaJxGIr0oamd9g"
+}' "https://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3/roles"
+```
+
+###### Response example
+
+```json
+{
+  "id": "irb1q92TFAHzySt3x0g4",
+  "role": "cr0WxyzJxGIr0ouum0g4",
+  "label": "UserCreatorRole",
+  "type": "CUSTOM",
+  "status": "ACTIVE",
+  "created": "2019-02-06T16:20:57.000Z",
+  "lastUpdated": "2019-02-06T16:20:57.000Z",
+  "assignmentType": "GROUP",
+  "resource-set": "iamoJDFKaJxGIr0oamd9g",
+  "_links": {
+    "assignee": {
+      "href": "http://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3"
+    },
+    "resource-set": {
+      "href": "http://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    },
+    "role": {
+      "href": "http://${yourOktaDomain}/api/v1/iam/roles/cr0Yq6IJxGIr0ouum0g3"
+    },
+    "permissions": {
+      "href": "http://${yourOktaDomain}/api/v1/iam/roles/cr0Yq6IJxGIr0ouum0g3/permissions"
+    }
+  }
+}
+```
+
+### Unassign a Role
+
+#### Unassign a Role from a User
+
+
+<ApiOperation method="delete" url="/api/v1/users/${userId}/roles/${roleId}" />
+
+Unassigns a Role from a User
+
+##### Request parameters
+
+
+| Parameter   | Description    | Param Type   | DataType   | Required |
+| :---------- | :------------- | :----------- | :--------- | :------- |
+| roleId      | `id` of a Role | URL          | String     | TRUE     |
+| userId      | `id` of a User | URL          | String     | TRUE     |
+
+##### Response parameters
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+##### Request example
+
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users/00u6fud33CXDPBXULRNG/roles/ra1b8anIk7rx7em7L0g4"
+```
+
+###### Response example
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Unassign a Role from a Group
+
+
+<ApiOperation method="delete" url="/api/v1/groups/${groupId}/roles/${roleId}" />
+
+Unassigns a Role from a Group
+
+##### Request parameters
+
+
+| Parameter       | Description       | Param Type   | DataType   | Required |
+| :-------------- | :---------------- | :----------- | :--------- | :------- |
+| groupId         | `id` of a Group   | URL          | String     | TRUE     |
+| roleId          | `id` of a Role    | URL          | String     | TRUE     |
+
+##### Response parameters
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+##### Request example
+
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3/roles/grasraHPx7i79ajaJ0g3"
+```
+
+##### Response example
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Unassign a Custom Role from a User or Group
```

Comment:
We need to be consistent: either each individual operation is marked as Beta, or the entire set of operations is marked as Beta. In its current form it's just confusing.